tests/src/ondevice_tests/crashdump_test.cpp

```
@@ -0,0 +1,61 @@
+#include <condition_variable>
+#include <mutex>
+#include <thread>
+
+#include "depthai/depthai.hpp"
+#include "utility/Environment.hpp"
+
+TEST_CASE("Crashdump callback is invoked on device crash") {
+    dai::utility::setEnv("DEPTHAI_CRASHDUMP", "0");  // don't save nor upload crash dump
+    dai::utility::setEnv("DEPTHAI_CRASH_DEVICE", "1");
+
+    dai::Device device;
```

Cppcheck (2.20.0)

[information] Include file

(missingIncludeSystem)